.gitignore

```
@@ -29,6 +29,7 @@ nohup.out
 # Hugo output
 public/
 resources/
+.hugo_bulid.lock
```

Nit: I think it should be 
```suggestion
.hugo_build.lock
```

---

Apologize for the confusion. I'll fix it right away.